src/configuration-store/configuration-store.ts

```diff
@@ -21,6 +21,8 @@ import { Environment } from '../interfaces';
  *
  * The policy choices surrounding the use of one or more underlying storages are
  * implementation specific and handled upstream.
+ *
+ * @deprecated To be replaced with ConfigurationStore and PersistentStorage.
```

Comment:
Are we not able to remove these entirely?

Reply:
We'll be able to once I finish precomputed client migration